src/Tizen.NUI.Scene3D/src/ModelView.cs

```diff
@@ -76,58 +70,74 @@ public ModelView(ModelView modelView) : this(Interop.ModelView.NewModelView(Mode
         }
 
         /// <summary>
-        /// Assignment operator.
+        /// Retrieves model root View.
         /// </summary>
-        /// <param name="modelView">Handle to an object.</param>
-        /// <returns>Reference to this.</returns>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public ModelView Assign(ModelView modelView)
+        public View ModelRoot
         {
-            ModelView ret = new ModelView(Interop.ModelView.ModelViewAssign(SwigCPtr, ModelView.getCPtr(modelView)), false);
-            if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
-            return ret;
+            get
+            {
+                View ret = new View(Interop.ModelView.GetModelRoot(SwigCPtr), false);
+                if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
+                return ret;
+            }
         }
 
         /// <summary>
-        /// Retrieves model root Actor.
+        /// Fits the model to the Control size.
         /// </summary>
-        /// <returns>Root View of the model.</returns>
         // This will be public opened after ACR done. (Before ACR, need to be hidden as Inhouse API)
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public View GetModelRoot()
+        public bool FitSize
```

Comment:
have more information on  property name would be better.
how about FitSizeOnControl ?
or what about making SizeMode enum and give an options like FitOnControl?

I understand the name came from DALi, so if we must maintain original name, and also it is just simple change method to property...  it is okay.

Reply:
Thank you I changed the name and added more comments.